core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,38 @@
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number} $width
```

Comment:
@wardpenney In thinking about this more, I think we should remove the default from this value. Iw we were to have a default, I think it would make sense to do something like have it be full width, ie 12/12.